core/tasks/channels/cron_test.go

```diff
@@ -0,0 +1,47 @@
+package channels_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/nyaruka/mailroom/core/tasks/channels"
+	"github.com/nyaruka/mailroom/testsuite"
+	"github.com/nyaruka/mailroom/testsuite/testdata"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSyncOldSeenAndroidChannels(t *testing.T) {
+	ctx, rt := testsuite.Runtime()
+
+	rt.Config.AndroidFCMServiceAccountFile = `testdata/android.json`
+	rc := rt.RP.Get()
+	defer rc.Close()
+
+	defer testsuite.Reset(testsuite.ResetAll)
+
+	mockFCM := testsuite.NewMockFCMService("FCMID3", "FCMID4", "FCMID5")
+	fc := mockFCM.GetClient(ctx)
+
+	// make all channel inactive
+	rt.DB.MustExec(`UPDATE channels_channel SET is_active = False WHERE id > 0`)
```

Comment:
can you rework this test to not modify the std test data? Then you can use `testsuite.Reset(testsuite.ResetData)` above which is a lot faster

Reply:
I am not following on this

Reply:
never mind I found that